.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules/
+dist/
```

Comment:
let's add a test to make sure you've ran `npm run build` and block PRs that did not run

Reply:
Done